vuepress/docs/next/docs/consume/custom-resources.md

```diff
@@ -86,12 +86,11 @@ cluster. After deploying this Keycloak instance, the Entando Operator will creat
 the necessary information for subsequent deployment operations to access the Keycloak instance as the Admin user. This
 allows the rest of the Entando Kubernetes Controllers to create a Keycloak OpenID Connect (OIDC) client for every HTTP service that
 gets deployed. If you already have a Keycloak instance that you want to use, you can skip this custom resource entirely
-and simply create the `keycloak-admin-secret' in the operator's namespace as specified in
-[this tutorial](../../tutorials/devops/external-id-management.md).
+and simply create the `keycloak-admin-secret' in the operator's namespace as specified in the 
```

Comment:
beware of apostrophes masquerading as backticks!!! secret' -> secret`

maybe do a search to make sure that one apostrophe caused all the blue text in this view??